app/views/subjects/index.html.haml

```diff
@@ -1,4 +1,8 @@
 %h1 Subjects
 %ul
+  - if Subject.count == 0
+    %li= 'No Subjects to Display.'
```

Comment:
This is probably over pedantic because we should have at lest on subject but whatever.